### DISCUSSION TITLE
Visual HTML Editor Addon for HTML Files

### DISCUSSION THREAD

**Discussion by jogibear9988** [microsoft/vscode-discussions#302] [Extension Development QnA] [answered]
Hello,
I'm working on a HTML Designer Framework in HTML (see: Demo: https://node-projects.github.io/web-component-designer-demo/index.html Code: https://github.com/node-projects/web-component-designer).

Now I wanted to create a VSCode Addon, wich could be used to edit HTML Files visually. Is this possible? So I don't want to create my own Text Editor instance inside of VSCode, I want a Addition window, wich is shown for a TextFile. So I also can write back to the HTML Code.

Maybe later this could be used for other files to, wich contain html Templates... like Vue Templates or other

Am I also be able to Access other Files form the loaded project? So I can for example import stylesheets?

**Comment by jogibear9988** [comment]
I know created a initial extension loading my designer:
https://github.com/node-projects/vs-code-designer-addon

but now I want this as a split view for a HTML document. Is this possible?
And when some changes something in the designer I want to write it back to the document (and also reload the designer when the html changes)

**Reply by burkeholland** [reply]
Is this still an open issue? Can you please respond with an update?

**Comment by jogibear9988** [comment, marked as answer]
No, the addon partialy works now. Split view works via opening the Document 2 times. Once in my designer, once in a normal texteditor.
